src/modules/ROOT/pages/changelog/202303/06.json

```diff
@@ -108,11 +97,22 @@
     },
     {
       "what": [
-        "false triggering of the unsaved changes message if a setting is enabled or disabled in {APP_NAME_ADMIN}'s Compose settings."
+        "triggering of unsaved changes message when trying to leave the Compose settings page in {APP_NAME_ADMIN} after making changes to the user interface settings."
```

Comment:
This was prompted even though the page was saved? Would be worth noting it down if that's the case.